### PR TITLE
Task/psgs 44 sub 54 jsons

### PR DESCRIPTION
**Pull request by maciejbak85**
Jsons copied from: https://input-output-hk.github.io/cardano-wallet/api/edge

Currently what I see we use configured dec/enc only when we need it ( for snake case ) and for the rest generic.auto is fine, 
so I used the same approach in tests.

### CODE REVIEW

File: src/test/java/iog/psg/cardano/jpi/JpiResponseCheck.java [hunk 1]
```diff
@@ -111,12 +114,24 @@ public static CardanoApi buildWithDummyApiExecutor() {
                             sync,
```
Comment (mcsherrylabs) [inline src/test/java/iog/psg/cardano/jpi/JpiResponseCheck.java:114]: I accidentally left 3 debug system.printlns in this class line 105. Could you remove please?

File: src/test/scala/iog/psg/cardano/util/ModelCompare.scala [hunk 2]
```diff
@@ -0,0 +1,136 @@
+package iog.psg.cardano.util
+
+import iog.psg.cardano.CardanoApiCodec._
+import org.scalatest.Assertion
+import org.scalatest.matchers.should.Matchers
+
+trait ModelCompare extends Matchers {
+
+  final def compareInAddress(decoded: InAddress, proper: InAddress): Assertion = {
```
Comment (mcsherrylabs) [inline src/test/scala/iog/psg/cardano/util/ModelCompare.scala:9]: Can you use 'expected' instead of 'proper'?

**Review by mcsherrylabs — APPROVED** [review]
I like it. It's tight. :+1:

File: src/test/scala/iog/psg/cardano/util/ModelCompare.scala (hunk 2 above)
Comment (maciejbak85) [reply src/test/scala/iog/psg/cardano/util/ModelCompare.scala:9]: yes sure

File: src/test/java/iog/psg/cardano/jpi/JpiResponseCheck.java (hunk 1 above)
Comment (maciejbak85) [reply src/test/java/iog/psg/cardano/jpi/JpiResponseCheck.java:114]: sure np.

**Review by mcsherrylabs — APPROVED** [review]

**Pull request merged**